app/gimlet/rev-b.toml

```diff
@@ -957,7 +957,8 @@ bus = "mid"
 address = 0x5a
 device = "raa229618"
 description = "CPU power controller"
-power = { rails = [ "VDD_VCORE", "VDD_MEM_ABCD" ] }
+power.rails = [ "VDD_VCORE", "VDD_MEM_ABCD" ]
```

Comment:
I'd be tempted to write this as
```toml
power.rails.VDD_CORE = [ 12, 13, 14, 15, 16, 17, 18, 19 ]
power.rails.VDD_MEM_ABCD = [ 0, 1, 2, 3 ]
```
Instead of having implicit pairs between `power.rails` and `power.phases`

or even 

```toml
power.rails.VDD_CORE.phases = [ 12, 13, 14, 15, 16, 17, 18, 19 ]
power.rails.VDD_MEM_ABCD.phases = [ 0, 1, 2, 3 ]
```

to be glaringly obvious about what's going on.

This would be a version archive bump and would require Humility changes to support, so it's understandable if you want to punt on it.

Reply:
Yes, all of this: really didn't want the archive bump right now.  (It's also true that most devices *don't* have explicit multiphase management -- and I don't want those devices to pay an abstraction tax for something that they will never support.)